### ISSUE TITLE
What does "I wish them great success in getting those valid needs met."

### ISSUE THREAD

**Issue by carun** [carun/mctb#2]
@danielmingram What does "those valid needs" imply here? Is this a sarcastic statement? Or is it referring them to do something else?

**Comment by danielmingram** [comment]
Yes, it could be read either sarcastically or non-sarcastically, left a touch ambiguous, to be honest, as, from a broader point of view, those are valid needs, and, from another, superficial needs in comparison to the deeper fruits of the Dharma. It is meant to be able to be read both ways.

**Comment by carun** [comment]
Thank you!
